src/include/distributed/distributed_planner.h

```diff
@@ -89,6 +92,12 @@ extern bool IsModifyDistributedPlan(struct DistributedPlan *distributedPlan);
 extern bool IsMultiTaskPlan(struct DistributedPlan *distributedPlan);
 extern bool IsMultiShardModifyPlan(struct DistributedPlan *distributedPlan);
 extern RangeTblEntry * RemoteScanRangeTableEntry(List *columnNameList);
+extern DeferredErrorMessage * RecursivelyPlanSubqueriesAndCTEs(Query *query,
```

Comment:
Ops, we've already defined them in `recursive_planner.h`

Reply:
Fixed.